app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletPresenter.kt

```diff
@@ -46,6 +68,41 @@ class WalletPresenter(
         }
     }
 
+    private fun loadStrigaOnRampTokens() {
+        launch {
+            val strigaOnRampTokens = strigaOnRampInteractor.getOnRampTokens().successOrNull().orEmpty()
+            viewStateFlow.emit(
+                viewStateFlow.value.copy(strigaOnRampTokens = strigaOnRampTokens)
+            )
+        }
+    }
+
+    private fun observeStrigaKycBanners() {
+        launch {
+            strigaUserInteractor.getUserStatusBannerFlow()
+                .collect { banner ->
+                    viewStateFlow.emit(
+                        viewStateFlow.value.copy(
+                            strigaBanner = banner
+                        )
+                    )
+                }
+        }
```

Comment:
strigaUserInteractor.getUserStatusBannerFlow()
            .map { viewStateFlow.value.copy(strigaBanner = it) }
            .collect { viewStateFlow.emit(it) }